docs/release_notes/10_0.md

```diff
@@ -0,0 +1,288 @@
+---
+title: Current Release 10
+tags:
+  - 10
+  - 10.0 release
+  - Rocky Linux 10
+---
+
+
+## Rocky Linux 10
+
+Rocky Linux 10 (RL 10) is the latest major release of Rocky Linux's enterprise operating system. 
+
+### New Features and Changes
+
+The following sections describe selected features of RL 10. 
+
+As with all major Rocky Linux releases, there are way too many enhancements, changes, and new features to go over in this release note. 
+
+The overview that follows highlights changes in the -  Architecture, Kernel, Installer, Networking, Security, Virtualization, Containers, Storage, Package management - categories. The features highlighted in this release note have been chosen based on their likely relevance to a general audience. 
+
+For more information, refer to the official upstream [Release Notes for Red Hat Enterprise Linux 10.0](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/10/html/10.0_release_notes/index). 
+
+### Supported Architectures
```

Comment:
We could reuse the same way to display that as for 9 back in the days, that looked pretty good 🙂  

https://docs.rockylinux.org/release_notes/9_0/#major-changes

And yeah `IBM Power Systems, Little Endian (POWER9)` and `64-bit IBM Z (z14)` are supported 👍🏻

Reply:
@sspencerwire did you miss the part about the how it's presented in 9?
Or intentional?